app/furniture/marketplace/_in_room_template.html.erb

```diff
@@ -0,0 +1 @@
+<%= turbo_frame_tag(furniture, src: furniture.location, data: { turbo_action: :advance }) %>
```

Comment:
Unfortunately, this has an annoying little flash every single time. I think what I want to do is extract the major substance of the `layouts/marketplace.html.erb` into a component or something and use it both here and in the layout.

Reply:
Flash how?

Reply:
Like, there is nothing there then *pow* it flashes into existence.